client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useDraftOutboundLines.tsx

```diff
@@ -73,11 +73,12 @@ export const useDraftOutboundLines = (
           const invoiceLine = lines?.find(
             ({ stockLine }) => stockLine?.id === batch.id
           );
-
           if (invoiceLine) {
             return createDraftStockOutLine({
               invoiceLine,
+              stockLine: batch,
```

Comment:
Wasn't 100% if this should be passed in or not, I think it wasn't included before to try to trigger the adjustment logic but there might have been another reason I'm not aware of too?